rott/vga8x16.h

```diff
@@ -0,0 +1,344 @@
+static uint8_t VGA8X16[4096] = {
```

Comment:
Where is this data from?

Reply:
<https://github.com/viler-int10h/vga-text-mode-fonts/blob/master/FONTS/PC-IBM/VGA8.F16>

Reply:
Hm, that's bad I'm afraid. This is a typical case of "found on the internet" without any copyright or license notice. 

Could you use the `textscreen/fonts/normal.h` file from Woof instead, with the accompanying copyright and license notice?

Reply:
What about <https://www.nongnu.org/vgabios/>? For the fonts, they claim that it's public domain. Apparently the BIOS font used by bochs/VirtualBox/qemu is based on the `fntcol16` package by Joseph Gil, who claimed them as public domain. This is all a mess, but I have no reason to expect that the DOSBox project did anything different.

The main reason I don't want to use the Woof font is because it would require me to change how the code works to place the font pixels :P

Reply:
Yes, okay. It's important if you take code from elsewhere that you can prove that you were explicitly allowed to take it. Availability itself does not suffice.

Reply:
Understood.